.vscode/cspell.json

```diff
@@ -178,7 +178,8 @@
     "westcentralus",
     "westus",
     "Wmem",
-    "xunit"
+    "xunit",
+    "Vlatest",
```

Comment:
Please move into a localized path for your library.   This is not a term that would be in the common set.